README.md

```diff
@@ -57,12 +57,38 @@ This project supports a basic [Vagrant](http://docs.vagrantup.com/v2/getting-sta
 
 1. Run vagrant up command
 
-    vagrant up
+        vagrant up
```

Comment:
I disagree. Indentation is the correct, most cross-compatible Markdown unless you need syntax highlighting.